kivymd/toast/kivytoast/kivytoast.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, **kwargs):
         self.add_widget(self.label_toast)
 
     def label_check_texture_size(
-        self, instance_label: Label, texture_size: list[int, int]
+        self, instance_label: Label, texture_size: Union[int, int]
```

Comment:
But `list` is incorrect, I get error

Reply:
Python 3.9.6 has a `Sequence`, but few people use this version of python. It can check arguments already directly in the function?

Reply:
No. Annotation of types only for programmers and IDEs

Reply:
But this causes an error(

Reply:
I am not getting any error ...

Reply:
Maybe List(...)

Reply:
Because you have python > 3.9

Reply:
> Maybe List(...)
? I think 
```
from typing import List
List[int]
``` It will be correct, although there will be no limit on the number of arguments

Reply:
It's just that you use python 3.9.6 typing, and this version is not worth much to anyone...

Reply:
I see no reason to support older versions of Python. And as far as I know, Python doesn't keep track of types. You can write whatever you want in annotations.

Reply:
Tracks, in python 3.8.5 I get this error - 
```
self, instance_label: Label, texture_size: list[int, int]
TypeError: 'type' object is not subscriptable
```